test/Dialect/FIRRTL/SFCTests/GrandCentralInterfaces/Companion.fir

```diff
@@ -1,4 +1,6 @@
-; RUN: firtool %s --grand-central-instantiate-companion | FileCheck %s --check-prefix CHECK
+; RUN: firtool %s --grand-central-companion-mode=instantiate | FileCheck %s --check-prefixes=INST
+; RUN: firtool %s --grand-central-companion-mode=drop | FileCheck %s --check-prefixes=DROP
+; RUN: firtool %s --grand-central-companion-mode=bind | FileCheck %s --check-prefixes=BIND
```

Comment:
The PR could be slightly more terse by using a common prefix, i.e., two prefixes per test to cover common cases like the YAML file output.

Reply:
There is an ordering problem that would make that ugly

Reply:
That's a good reason to not do it! Thanks for checking if it was possible.